main.go

```diff
@@ -259,12 +260,24 @@ func main() {
 		fatal.Exit("gehen.yml must contain at least one service")
 	}
 
-	// Connect to ECS API
 	sess := session.Must(session.NewSession(&aws.Config{
 		Region: aws.String("us-east-1"),
 	}))
-	ecsClient := ecs.New(sess)
-	ebClient := eventbridge.New(sess)
+
+	// Connect to ECS and EventBridge APIs
+	var ecsClient *ecs.ECS
+	var ebClient *eventbridge.EventBridge
+
+	if role != nil && role.AccountID != "" && role.Name != "" {
```

Comment:
I could also make config return an error if a role was parsed that had one of AccountID and Name being its zero value, will think more tomorrow my brain is busted rn